### ISSUE TITLE
Memory View display refresh issue

### ISSUE THREAD

**Issue by chostelet** [uxmal/reko#577]
Assuming you have a segment of memory whose starting load address is not  multiple of 16, then when you browse in the memory view using the mouse wheel (scroll-down, scroll-up), then you can't display the first line. again. Except if you click on the up-arrow of the vertical scrollbar. Only full lines (16 bytes long) are visible navigating with the mouse wheel.

**Comment by smx-smx** [comment]
Yes your issue is more specific to the boundary of the segment. It's very likely because the start of the segment is not 16 bytes aligned but, apparently, the memory viewer has a 16 bytes granularity when scrolling.
Once it gets aligned it won't show the initial unaligned line anymore

**Comment by chostelet** [comment]
I've the feeling that the new title is bit biased. :-).
The example I provided is showing a segment which is aligned. it is word-aligned...
Why is the memory viewer assuming (forcing?) anything about memory segments alignment?
The display can show some alignments (per user's reference on the number of bytes per line) but it should not assume the memory segments are aligned the way it "thinks" of alignment...
So to summarize, the issue is not with the boundaries of the segment, it is with the viewer. :-)

**Comment by smx-smx** [comment]
I meant 16 byes alignment in the title. By the way it's probably related to this: https://github.com/uxmal/reko/blob/master/src/Gui/Windows/Controls/MixedCodeDataModel.cs#L132
